microcosm_flask/conventions/relation.py

```diff
@@ -41,11 +42,56 @@ def configure_createfor(self, ns, definition):
         @response(definition.response_schema)
         def create(**path_data):
             request_data = load_request_data(definition.request_schema)
-            response_data = definition.func(**merge_data(path_data, request_data))
+            response_data = require_response_data(definition.func(**merge_data(path_data, request_data)))
```

Comment:
@jessemyers  I noticed this decorator is now used for all the main CRUD operators, thought it should also be applied here for the 'For' versions?